### ISSUE TITLE
zone/nation enter/exit message config?

### ISSUE THREAD

**Issue by ghost** [Arckenver/Nations#108]
i am trying to make plot lend system by using Natinos plugin
it seems work properly but when im moving in 'plot land', system keep spamming because of enter/exit messages
it is like -
"PlotLand-PlotA-PlayerA-(No PvP)"
"PlotLand-PlotB-PlayerA-(No PvP)" (PlotB is next to PlotA)
"PlotLand-(No PvP)"
"PlotLand-PlotC-PlayerB-(No PvP)"
"PlotLand-(No PvP)"
"PlotLand-PlotB-PlayerA-(No PvP)"
"PlotLand-PlotA-PlayerA-(No PvP)"

i am planning to make about 100plots per 1 plot land
but i think this spamming problem will make players annoying
thanks for make this cool plugin though :)

**Comment by TheoKah** [comment]
I'll add an option in config to mute the message when entering/leaving a zone and/or a nation

**Comment by turtledude01** [comment]
I personally think a better approach to this would be to display the enter/leave msg in the title (small text in the middle of the screen) the same way as the text that is displayed when you try to use a bed during day.

**Comment by TheoKah** [comment]
The title/subtitle would still spam the player...

I would have consider the actiontitle (the really small one above xp) but it's minecraft 1.11

Maybe I can use the title/subtitle when entering leaving a nation, but not a zone

**Comment by turtledude01** [comment]
It technicaly would spam the player, but it is way less spam and easier to ignore than the chat being blown up with enter/leave msges.  I have tried it and what happens is if you change regions very quickly it just overrites the current title/subtitle (im setting the subtitle to be literally the same msg as what is displayed in chat normally)

**Comment by TheoKah** [comment]
I'll make a few tests then

**Comment by turtledude01** [comment]
Example of what I'm doing: 
```
if (ConfigHandler.getNode("others", "useTitleForNationEnter").getBoolean())
		{
			player.sendMessage(ChatTypes.ACTION_BAR, builder.build());
		}
		else
		{
			player.sendMessage(builder.build());
		}
```

**Comment by TheoKah** [comment]
Isn't the actionbar a 1.11 feature? If not then this is what I'll use.

**Comment by turtledude01** [comment]
I'm not 100% sure what version it is in, but I do know it is available in the sponge API your building against.  Even so, adding a few options for it wouldn't be a bad idea.

**Comment by turtledude01** [comment]
I just checked and it is in fact 1.11 but a config option to use action bar, and an option to disable messages all together would be enough for any version to be able to use it with less spam.

**Comment by TheoKah** [comment]
This will be available in v2.5 later today
